Assets/MXR.SDK/Runtime/Types/StatusTypes.cs

```diff
@@ -122,6 +126,40 @@ public class Controller {
         public string version = string.Empty;
     }
 
+    /// <summary>
+    /// Enables tracking and reporting of network errors and connectivity events across different SSIDs on a device.
+    /// </summary>
+    [System.Serializable]
+    public class NetworkErrorCodeFrequency {
+        public List<NetworkErrorStatistics> networkErrorStatistics = new List<NetworkErrorStatistics>();
+        public int connectivityFlippedCount = 0;
+    }
+
+
+    /// <summary>
+    /// Stores statistics related to network errors for a specific endpoint.
+    /// </summary>
+    [System.Serializable]
+    public class NetworkErrorStatistics {
+        public string endPointName;
```

Comment:
In Admin app, this is "endpointName". Maybe the JSON serialization library is saving you from the horrors of case sensitivity, or maybe this is the cause of a bug - not sure if you've tested this yourself yet.